packages/contracts/contracts/ChugSplashManager.sol

```diff
@@ -145,6 +146,11 @@ contract ChugSplashManager is Owned {
      */
     bytes32 public activeBundleId;
 
+    /**
+     * @notice The address of the Executor Selection Strategy for this project.
+     */
+    address internal executorSelectionStrategy;
```

Comment:
Minor modification, prefer that this be `IExecutorSelectionStrategy public ess`. Gives better type safety and means we don't need to cast stuff in other places.